extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/config/GreenbidsRealTimeDataConfiguration.java

```diff
@@ -0,0 +1,124 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.config;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.maxmind.geoip2.DatabaseReader;
+import io.vertx.core.Vertx;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.core.ThrottlingThresholds;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.config.GreenbidsRealTimeDataProperties;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.data.GreenbidsInferenceDataService;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.FilterService;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.ModelCache;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.OnnxModelRunner;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.OnnxModelRunnerWithThresholds;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.ThresholdCache;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.result.GreenbidsInvocationService;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.v1.GreenbidsRealTimeDataModule;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.v1.GreenbidsRealTimeDataProcessedAuctionRequestHook;
+import org.prebid.server.json.ObjectMapperProvider;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@ConditionalOnProperty(prefix = "hooks." + GreenbidsRealTimeDataModule.CODE, name = "enabled", havingValue = "true")
+@Configuration
+@EnableConfigurationProperties(GreenbidsRealTimeDataProperties.class)
+public class GreenbidsRealTimeDataConfiguration {
+
+    @Bean
+    public DatabaseReader dbReader(GreenbidsRealTimeDataProperties properties) {
+        final File database = new File(properties.getGeoLiteCountryPath());
+        try {
+            return new DatabaseReader.Builder(database).build();
+        } catch (IOException e) {
+            throw new PreBidException("Failed build DatabaseReader from DB", e);
+        }
+    }
+
+    @Bean
+    public GreenbidsInferenceDataService greenbidsInferenceDataService(DatabaseReader dbReader) {
+        return new GreenbidsInferenceDataService(dbReader, ObjectMapperProvider.mapper());
+    }
+
+    @Bean
+    GreenbidsRealTimeDataModule greenbidsRealTimeDataModule(
```

Comment:
Please be consistent in defining beans. As I see elsewhere in the project, we don't usually use the `public` modifier for this.